packages/mask/src/extension/popups/pages/Personas/ConnectedWallets/index.tsx

```diff
@@ -32,7 +33,7 @@ const ConnectedWallets = memo(() => {
         const results = await Promise.all(
             proofs.map(async (x) => {
                 if (x.platform === NextIDPlatform.Ethereum) {
-                    const domain = await NameService?.reverse?.(chainId, x.identity)
+                    const domain = await NameService?.reverse?.(ChainId.Mainnet, x.identity)
```

Comment:
Dead code the chain id in `/ENS.ts`. Keep the UI part compatible with all networks.